modules/targets/darwin/options.nix

```diff
@@ -0,0 +1,182 @@
+{ config, lib, ... }:
+
+with lib;
+
+let
+  mkOption = args:
+    lib.mkOption (args // {
+      type = types.nullOr args.type;
+      default = null;
+    });
+
+  mkEnableOption = name:
+    lib.mkOption {
+      type = with types; nullOr bool;
+      default = null;
+      example = true;
+      description = "Whether to enable ${name}.";
+    };
+
+  safari = config."com.apple.Safari";
+in {
+  freeformType = with types; attrsOf attrs;
```

Comment:
`attrsOf anything`?
`attrsOf attrs` is unchecked and unevaluated (no merging).

Reply:
Fixed. It's now `attrsOf (attrsOf anything)`.